training/training.module

```diff
@@ -0,0 +1,378 @@
+<?php
+
```

Comment:
add @file tag with a description at the very beginning of the file
see https://www.drupal.org/coding-standards/docs#file